lib/src/tree.dart

```diff
@@ -676,7 +679,7 @@ class MixinDefinition extends Directive {
 
 /** Support a Sass @mixin. See http://sass-lang.com for description. */
 class MixinRulesetDirective extends MixinDefinition {
-  final List<RuleSet> rulesets;
+  final List rulesets;
```

Comment:
any reason we can't preserve the type arg here? or was the list used with a mix of types not just RuleSet?